Dockerfile

```diff
@@ -1,61 +1,72 @@
-FROM fedora
+FROM alpine:3.5
 
-MAINTAINER Guillaume Scheibel <guillaume.scheibel@gmail.com>
+LABEL MAINTAINER="Guillaume Scheibel <guillaume.scheibel@gmail.com>"
+LABEL MAINTAINER="Damien DUPORTAL <damien.duportal@gmail.com>"
 
-ENV JAVA_HOME /jdk1.8.0_112
-ENV PATH $PATH:$JAVA_HOME/bin:/fopub/bin
-ENV BACKENDS /asciidoctor-backends
-ENV GVM_AUTO_ANSWER true
-ENV ASCIIDOCTOR_VERSION "1.5.5"
+ENV JAVA_HOME=/usr/lib/jvm/default-jvm \
+  PATH=${PATH}:${JAVA_HOME}/bin:/fopub/bin \
+  BACKENDS=/asciidoctor-backends \
+  GVM_AUTO_ANSWER=true \
+  ASCIIDOCTOR_VERSION="1.5.5"
 
-RUN dnf install -y tar \
-    make \
-    gcc \
-    ruby \
-    ruby-devel \
-    rubygems \
+RUN apk --update --no-cache add \
+    asciidoctor \
+    bash \
+    build-base \
+    curl \
     graphviz \
-    rubygem-nokogiri \
+    jpeg \
+    jpeg-dev \
+    openjdk8 \
```

Comment:
AFAIK, this come from fopub tool: https://github.com/asciidoctor/asciidoctor-fopub#prerequisites . But given @mojavelinux comment below (https://github.com/asciidoctor/docker-asciidoctor/pull/39#issuecomment-288642823 ) , this would be a great move if we got out the jdk to the jre (even not using java at all in this image)